src/data/nav-items.yaml

```diff
@@ -8,6 +8,12 @@
       path: /all-about-carbon/the-team
     - title: Partners
       path: /all-about-carbon/partners
+- title: Case studies
```

Comment:
This is the shorter-term solution, right - as @janchild and I discussed earlier this week. Longer term we might look at a different way to present case studies within the nav.

Reply:
Yep! It's just a stand in navigation until we have a better idea of what the longer term solution would be.